.gitignore

```diff
@@ -7,22 +7,12 @@ dist
 
 # cartopy build files
 lib/cartopy/_version.py
-lib/cartopy/trace.cpp
-lib/cartopy/trace.so
-lib/cartopy/trace.*.so
-lib/cartopy/trace.pyd
```

Comment:
There should be `pyd` files on Windows?

Reply:
Ahh, yes. Thanks, just added `*.pyd` to the gitignore.